assets/src/containers/Course.js

```diff
@@ -23,7 +24,6 @@ function Course (props) {
     }
     setSideDrawerState(open)
   }
-
```

Comment:
Might be some value to having this newline to separate the `error` checks from the `toggleDrawer` event handler.

Reply:
added the new line back.

Reply:
added the new line back.